spark/src/main/scala/geotrellis/spark/costdistance/IterativeCostDistance.scala

```diff
@@ -52,6 +50,27 @@ object IterativeCostDistance {
 
   val logger = Logger.getLogger(IterativeCostDistance.getClass)
 
+  /**
+    * Compute the resolution (in meters per pixel) of a layer.
+    */
+  private [spark] def computeResolution[K: (* => SpatialKey), V: (* => Tile)](
```

Comment:
I decided to move this function into the `CostDistance` object to avoid code duplication (we had two versions of the same function implemented in two different places).